### DISCUSSION TITLE
Music Assistant Ingress page Reload Period

### DISCUSSION THREAD

**Discussion by rfporter** [music-assistant/support#3238] [Q&A] [answered]
When Music Assistant is accessed from the Sidebar of Home Assistant, the Music Assistant page is reloaded every 30 seconds. This is ok when you are just clicking around the music collection, but becomes quite annoying when you are trying to configure a provider or player and haven't clicked the save button yet.

Is this refresh every 30 seconds required? Can the period be adjusted to, lets say, 5 minutes instead?

**Comment by OzGav** [comment]
That isn't normal.

**Reply by EddyBurnett** [reply]
Same here only with Home Assistant in https (HA Proxy) and only in the Music Assistant menu, All other Home Assistant parts (like Node-RED,  front-end etc) are fine.
When accessing Home Assistant directly in http then this doesn't happen.

**Reply by rfporter** [reply]
I've got the same setup. https home assistant using HA Proxy. Only music assistant page reloads every 30s.

The reload issue doesn't happen when accessing home assistant directly.

**Comment by EddyBurnett** [comment, marked as answer]
Ok found the issue. Has to do with the websockets connection, although weird that I didn't had issues before with other Home Assistant stuff.

Just add this to your HAProxy Backend server pool:

**Advanced Settings -> Backend pass thru**

```
option http-server-close
timeout tunnel 1h
```

That fixed it for me...

**Reply by rfporter** [reply]
That fixed it for me too. Thanks!

**Reply by purpletortue** [reply]
Same here, thank you
